i18n/ja.json

```diff
@@ -48,10 +48,6 @@
     "translation": "9月"
   },
   {
-    "id": "api.admin.add_certificate.array.app_error",
```

Comment:
This id is deleted in all languages.
But this id is used in [platform/admin\.go at 1e7985a87a72bea9a308cf1506dacc828c6e2e1c · mattermost/platform](https://github.com/mattermost/platform/blob/1e7985a87a72bea9a308cf1506dacc828c6e2e1c/api/admin.go#L661).

Is this ok?